unity-renderer/Assets/DCLPlugins/EquippedEmotes/EquippedEmotesPlugin.cs

```diff
@@ -53,9 +75,13 @@ internal void LoadEquippedEmotesFromLocalStorage()
             SetEquippedEmotes(storedEquippedEmotes);
         }
 
-        internal void SaveEquippedEmotesInLocalStorage(IEnumerable<EquippedEmoteData> equippedEmotes)
+        internal void OnEquippedEmotesSet(IEnumerable<EquippedEmoteData> equippedEmotes) { SaveEquippedEmotesInLocalStorage(); }
+
+        internal void OnEquippedEmoteAddedOrRemoved(EquippedEmoteData equippedEmote) { SaveEquippedEmotesInLocalStorage(); }
+
+        internal void SaveEquippedEmotesInLocalStorage()
         {
-            if (!emotesCustomizationDataStore.isEmotesCustomizationInitialized.Get())
+            if (!featureFlagsDataStore.flags.Get().IsFeatureEnabled(EMOTES_CUSTOMIZATION_FEATURE_FLAG))
```

Comment:
Do we need this? We will never receive changes on the equipped emotes without the feature falg because the UI won't be available.

Reply:
You're right. It is no longer needed. Done in https://github.com/decentraland/unity-renderer/pull/2078/commits/f621f954f50177a7fc2405c64b4f155cc11245d3!